ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -64,21 +78,60 @@ internal fun TipsTutorialLayout() = GodToolsTheme() {
         LottieAnimation(
             composition,
             { progress },
-            modifier = Modifier.height(290.dp).fillMaxWidth()
+            modifier = Modifier
+                .height(290.dp)
+                .fillMaxWidth()
         )
 
         Text(
-            text = stringResource(R.string.tutorial_tips_learn_headline),
+            text = stringResource(title),
             style = MaterialTheme.typography.titleLarge,
             textAlign = TextAlign.Center,
 
         )
 
         Text(
-            stringResource(R.string.tutorial_tips_learn_text),
-            style = MaterialTheme.typography.bodyLarge,
+            stringResource(body),
+            style = MaterialTheme.typography.bodyMedium,
             textAlign = TextAlign.Center,
             modifier = Modifier.padding(top = 16.dp)
         )
+
+        if (body2 != null) {
+
+            Text(
+                // Spacer(modifier = Modifier.minLinesHeight(minLines = 1, textStyle = MaterialTheme.typography.bodyMedium)),
+                stringResource(body2),
+                style = MaterialTheme.typography.bodyMedium,
+                textAlign = TextAlign.Center
+            )
+        }
+        if (anim == R.raw.anim_tutorial_tips_light) {
```

Comment:
this conditional should be branched based on which `Page` you are rendering, and not based on which animation is used. This will be possible once you pass the `Page` object into `TipsTutorialLayout()`